src/main/java/com/minecolonies/coremod/colony/colonyEvents/citizenEvents/CitizenGrownUpEvent.java

```diff
@@ -0,0 +1,59 @@
+package com.minecolonies.coremod.colony.colonyEvents.citizenEvents;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.minecolonies.api.util.constant.Constants;
```

Comment:
also here

Reply:
do you mean the empty line?
if yes, thats how eclipse formats imports, i can remove it if you want.

Reply:
nvm, it was a github problem